.vsts-ci.yml

```diff
@@ -411,7 +411,7 @@ stages:
     jobs:
     - job: Copy_SDK_To_Latest
       pool:
-        name: Hosted VS2017
+        image: windows-2019
```

Comment:
I'm not familiar with using `image` instead of `vmimage`, does that also work?

Reply:
The pool schema: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pool says it should be `vmimage` but azure devops has aliases for many things so I'm not sure if this also works

Reply:
typo, let me fix